nuxeo-services/nuxeo-platform-web-common/src/main/java/org/nuxeo/ecm/platform/ui/web/download/DownloadServlet.java

```diff
@@ -60,7 +51,7 @@
 
     private static final long serialVersionUID = 1L;
 
-    private static final Log log = LogFactory.getLog(DownloadServlet.class);
+    protected DownloadService downloadService = Framework.getService(DownloadService.class);
```

Comment:
Never store a framework service in a field, always look it up just when needed. This is so that we can maybe one day have proper hot-reload everywhere.